test-all-servers

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# declare an array with all servers
+array=( eu-ger-1 eu-ger-2 eu-fin-1 us-or-1 us-or-2 us-pa-1 us-pa-2 us-va-1 )
+
+for SERVER in "${array[@]}"
+do
+	SKYNET_JS_INTEGRATION_TEST_SERVER="https://$SERVER.siasky.net" yarn test src/integration.test.ts
+done
```

Comment:
You should use matrix strategy for listing test servers instead of this manual loop.

https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix

Reply:
From what i understand we want to run this script from the healthcheck as well.

Reply:
I don't think there is any reason to do so, health checks run on each server and test only that specific server's health, we don't want to run a full test suite against all servers from every single server

Reply:
Okay, I'll remove this script and make a follow-up issue for the healthcheck

Reply:
See https://github.com/NebulousLabs/skynet-webportal/issues/598